serverless/db/mongo/models/sagawa/index.js

```diff
@@ -509,12 +509,10 @@ export default (db) => {
         JWT_SECRET: jwtSecret,
         LAMBDA_ENV: lambdaEnv,
         BASE_URL: baseUrl,
-        PRODUCTION_URL: productionUrl,
       } = process.env;
 
       const token = JWT.sign(payload, jwtSecret);
-      const prodEnv = lambdaEnv === 'production';
-      const trackingLink = `${prodEnv ? `${productionUrl}/#!` : `${baseUrl}/#!`}/tracking?token=${token}`;
+      const trackingLink = `${baseUrl}/tracking?token=${token}`;
```

Comment:
you need to add `/#!` so it should look like `${baseUrl}/#!/tracking?token=${token}`

Reply:
Nice Catch!!

Reply:
Fixed it!!